pages/k8s/install-manual.md

```diff
@@ -3,112 +3,209 @@ wrapper_template: "base_docs.html"
+   <span>Enables native support such as EBS storage and ELB load balancers. <a href="#"> Read more...</a></span>
+  </div>
+  <div class="col-4 ">
+    <span><a href="https://raw.githubusercontent.com/charmed-kubernetes/bundle/master/overlays/aws-overlay.yaml" class="p-button--positive">Download aws-overlay.yaml</a></span>
+  </div>
+</div>
+<br>
+<div class="row">
+<div class="col-2 u-vertically-center">
+  <span>Azure integrator</span>
+</div>
+ <div class="col-6 u-vertically-center">
+  <span>Enables support for Azure's native Disk Storage and load balancers.</span>
+ </div>
+ <div class="col-4 u-vertically-center">
+   <span class="u-vertically-center"><a href="https://raw.githubusercontent.com/charmed-kubernetes/bundle/master/overlays/azure-overlay.yaml" class="p-button--positive">Download azure-overlay.yaml</a></span>
+ </div>
+</div>
+<br>
+<div class="row">
+<div class="col-2">
+  <span>GCP integrator</span>
+</div>
+ <div class="col-6">
+  <span>Integrates with GCP for storage and loadbalancing. <a href="#"> Read more...</a></span>
+ </div>
+ <div class="col-4">
+   <span class="u-vertically-center"><a href="https://raw.githubusercontent.com/charmed-kubernetes/bundle/master/overlays/gcp-overlay.yaml" class="p-button--positive">Download gcp-overlay.yaml</a></span>
+ </div>
+</div>
+<br>
+ <div class="row">
+  <div class="col-2">
+   <span>OpenStack integrator</span>
+ </div>
+  <div class="col-6">
+   <span>Provides support for OpenStack native features such as Cinder volumes and LBaaS. <a href="#"> Read more...</a></span>
+  </div>
+  <div class="col-4">
+    <span><a href="https://raw.githubusercontent.com/charmed-kubernetes/bundle/master/overlays/openstack-overlay.yaml" class="p-button--positive u-no-margin--right">Download openstack-overlay.yaml</a></span>
+  </div>
+</div>
+<br>
+ <div class="row">
+  <div class="col-2">
+   <span>vSphere integrator</span>
+ </div>
+  <div class="col-6">
+   <span>Provides support native storage in vSphere. <a href="#"> Read more...</a></span>
+  </div>
+  <div class="col-4">
+    <span><a href="https://raw.githubusercontent.com/charmed-kubernetes/bundle/master/overlays/vsphere-overlay.yaml" class="p-button--positive u-no-margin--right u-no-margin--left">Download vsphere-overlay.yaml</a></span>
+  </div>
+</div>
+</div>
 
-### Create a model
 
-The controller automatically creates a model named 'default'. It is useful to
-have a different model for each deployment of **CDK**, and for the models to have
-useful names. You can create a new model with the `add-model` command like
-this:
 
-```bash
-juju add-model cdk-test
-```
 
-...susbstituting the desired name.
 
-### Deploy the CDK bundle
 
-The **Juju Charm Store** hosts the **CDK** bundles as well as individual charms.  To
-deploy the latest, stable bundle, run the command:
+You can use multiple overlays (of different types) if required.  Note that all
+the 'integrator' charms require the use of the `--trust` option. For example,
```

Comment:
Is that still true with trust being inside the overlay yaml?

Reply:
Yes. Otherwise Juju just ignores it. According to the docs, it's supposed to report an error, but when I tried it just did nothing